models/public/face-recognition-mobilefacenet-arcface/face-recognition-mobilefacenet-arcface.md

```diff
@@ -0,0 +1,68 @@
+# face-recognition-mobilefacenet-arcface
+
+## Use Case and High-Level Description
+
+[Deep face recognition net with MobileFaceNet backbone and Arcface loss](https://arxiv.org/abs/1801.07698)
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Face recognition |
+| GFLOPs            | 0.449         |
+| MParams           | 0.993         |
+| Source framework  | mxnet\*       |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `data`,  shape - `1,3,112,112`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
```

Comment:
By default MXnet uses images in `RGB` channel order. Please, prove me wrong.

Reply:
Correct, thanks